lambda_layers/core_layer/python/core_layer/model/item_model.py

```diff
@@ -63,6 +63,18 @@ def to_dict(self, with_tags=False, include_type=False):
                 tags_list.append(item_tag.tag.tag)
             item_dict["tags"] = tags_list
 
+        if with_urls:
+            url_list = []
+            for item_url in self.urls:
+                url_dict = {
+                    'url': item_url.url.url,
+                    # TODO: Replace with is_safe variable, once available
+                    'is_safe': True
```

Comment:
Should we name it positive (is_safe) or negative (unsafe)?

Normally I prefer positive naming. 
But right now I have the following structure in my "safe-url-check" branch.
Item#status = "Unsafe"
ItemUrl#unsafe = "GOOGLE:MALWARE"
Url#unsafe = "GOOGLE:MALWARE"
The string "unsafe" stores the extracted threat of the URL or None if everything is ok.

I think we could leave it as it is and just translate the negative naming to the "is_safe" here. What do you think?

Reply:
Yes, I completely agree. I think the frontend doesn't need to know the exact threat, but only whether the link is safe or not, so translating it at this point makes sense!